app/gui2/ydoc-server/languageServerSession.ts

```diff
@@ -503,7 +503,10 @@ class ModulePersistence extends ObservableV2<{ removed: () => void }> {
         const reloading = this.ls.closeTextFile(this.path).then(() => {
           return this.ls.openTextFile(this.path)
         })
-        this.lastAction = reloading.then()
+        this.lastAction = reloading.then(
+          () => {},
+          () => {},
+        )
```

Comment:
Can we at least write a log here? Discarding errors silently is usually a very bad idea.

If there is some bizarre reason to ignore it completely, it's worth mentioning here in a comment.